utils/helpers.py

```
@@ -0,0 +1,190 @@
+    """
+    Format duration in seconds to human-readable string.
+    
+    Args:
+        seconds: Duration in seconds
+        
+    Returns:
+        Formatted duration string
+    """
+    if seconds < 60:
+        return f"{seconds:.2f}s"
+    elif seconds < 3600:
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)